tests/hawc/apps/assessment/test_view_permissions.py

```diff
@@ -20,6 +20,7 @@
     "team_members_1": ("1", "2"),
     "reviewers_0": ("1",),
     "reviewers_1": ("1",),
+    "epi_version": ("1",),
```

Comment:
can you add to the `tests/data/fixtures.db.yaml` the epi_version: 1 for all the assessments there as well?